samples/client/petstore/python-experimental/tests/test_deserialization.py

```diff
@@ -377,6 +380,36 @@ def get_header(name, default=None):
         finally:
             os.unlink(file_path)
 
+    def test_deserialize_binary(self):
```

Comment:
Can you rename this deserialize_binary_to_str ?
The current name implies that we expect to receive binary data and process it as binary in downstream consumers.

Reply:
Can you add a separate test where we keep the response data in bytes?
I'm not sure if we should return an open file object if bytes are received we chose to store them in a file object before because a file was sent. How about keeping non-file bytes in a value of type bytes?
If I were receiving bytes as a user I would not expect them to be stored in a file object.

Reply:
How do we know that sending bytes will work?